streams/src/test/java/org/apache/kafka/streams/state/internals/TimestampedWindowStoreBuilderTest.java

```diff
@@ -223,38 +208,9 @@ public void shouldThrowNullPointerIfInnerIsNull() {
         assertThrows(NullPointerException.class, () -> new TimestampedWindowStoreBuilder<>(null, Serdes.String(), Serdes.String(), new MockTime()));
     }
 
-    @Test
-    public void shouldThrowNullPointerIfKeySerdeIsNull() {
-        assertThrows(NullPointerException.class, () -> new TimestampedWindowStoreBuilder<>(supplier, null, Serdes.String(), new MockTime()));
-    }
-
-    @Test
-    public void shouldThrowNullPointerIfValueSerdeIsNull() {
```

Comment:
Hmm...this looks familiar. I remember reviewing another PR which had similar tests.

Could you please comment here why the tests were passing earlier and what is the right expectation (and why?).

Reply:
This looks like the same issue as https://github.com/apache/kafka/pull/14152#discussion_r1284245654

Reply:
Yes. that is the PR I was thinking of. Thank you for pointing it to me.